Src/IronPython/Runtime/Bytes.cs

```diff
@@ -16,6 +16,7 @@
 
 using IronPython.Runtime.Operations;
 using IronPython.Runtime.Types;
+using System.Text;
 
 namespace IronPython.Runtime {
     [PythonType("bytes")]
```

Comment:
Looks like you'll need to add `Serializable` to `Bytes` for .NET Framework since `Exception.Data` elements need to be serializable.

Reply:
it just shows the necessity of always testing on Windows